src/bun.zig

```diff
@@ -7,6 +7,8 @@
 const bun = @This();
 
 pub const Environment = @import("./env.zig");
+pub const EnvVar = @import("./env_var.zig");
```

Comment:
Would it be too inconsistent to name the variable `env_var`? Really, `Environment`, `EnvVar`, and `FeatureFlag` should all be lowercase. Maybe it would be too weird to just change this one, but I just wish we could stick to the few conventions Zig prescribes.